corehq/util/spreadsheets/excel_importer.py

```diff
@@ -17,6 +17,8 @@ def __init__(self, task, file_ref_id):
 
         download_ref = DownloadBase.get(file_ref_id)
         self.workbook = WorkbookJSONReader(download_ref.get_filename())
+        for ws in self.workbook.worksheets:
```

Comment:
wow, good catch.  Maybe this should be encapsulated in ```WorkbookJSONReader``` to make sure that the next person who uses the library won't run into the same error.